### ISSUE TITLE
Deep Unordered Composition Rivals Syntactic Methods for Text Classification

### ISSUE THREAD

**Issue by mingqook** [mingqook/Reading_Paper#19]
## Abstract (초록) 📝
Many existing deep learning models for natural language processing tasks focus on learning the compositionality of their inputs, which requires many expensive computations. We present a simple deep neural network that competes with and, in some cases, outperforms such models on sentiment analysis and factoid question answering tasks while taking only a fraction of the training time. While our model is syntactically-ignorant, we show significant improvements over previous bag-of-words models by deepening our network and applying a novel variant of dropout. Moreover, our model performs better than syntactic models on datasets with high syntactic variance. We show that our model makes similar errors to syntactically-aware models, indicating that for the tasks we consider, nonlinearly transforming the input is more important than tailoring a network to incorporate word order and syntax.

## 논문(Paper)에서 다루는 핵심 개념 (Idea) 💡
* DAN(Deep Averaging Network), Unordered composition, Syntatic composition, Word dropout

## 논문(Paper) 정리 
* https://velog.io/@mingqook/Deep-Unordered-Composition-Rivals-Syntactic-Methodsfor-Text-Classification

## 논문(Paper) 원본
* https://aclanthology.org/P15-1162/
